recipes/pynfft/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.3.2" %}
+
+package:
+  name: pynfft
+  version: {{ version }}
+
+source:
+  fn: pyNFFT-{{ version }}.tar.bz2
+  url: https://pypi.io/packages/source/p/pyNFFT/pyNFFT-{{ version }}.tar.gz
+  md5: d2457a3f43839a8acce3295c60d5e692
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - numpy >=1.6
+    - python >=2.7
+    - setuptools
+    - nfft >=3.2,<3.3
+    - gcc
+    - cython
+  run:
+    - numpy >=1.6
```

Comment:
If https://github.com/conda-forge/staged-recipes/pull/583/files#r62758763 is true than you need to use `numpy x.x` here too.